apps/public-docsite-v9/src/Concepts/Positioning/PositioningShiftToCoverTarget.stories.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33468&commitId=68845161bf481a928abae9d5bd5f49ba3634db86&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|9| Changed|
|Avatar Converged.badgeMask.chromium.png|28| Changed|

</details>


  <details><summary><b>Drawer</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay RTL.chromium.png|1167| Changed|
|Drawer.Full Overlay High Contrast.chromium.png|2244| Changed|
|Drawer.Full Overlay Dark Mode.chromium.png|2798| Changed|

</details>


  <details><summary><b>Positioning</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Positioning.shiftToCoverTarget with autoSize and async content.chromium.png|0| Added|
|Positioning.shiftToCoverTarget with autoSize.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>